### ISSUE TITLE
[Follower node crash] Unable to sync burnchain: Invalid argument (os error 22)

### ISSUE THREAD

**Issue by zone117x** [blockstack/stacks-blockchain#1488]
**Describe the bug**
Had two separate follower nodes crash at around block 410.

(See full debug log in comment below)
```
stacks-blockchain_1  | INFO [1587722469.382] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/chat.rs:1067] convo:id=114,outbound=true,peer=facade01+80000000://V4(35.245.47.179:20444): failed to recv on P2P conversation: PermanentlyDrained
stacks-blockchain_1  | INFO [1587722469.383] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:449] No open socket for 114
stacks-blockchain_1  | WARN [1587722491.264] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:942] Failed to get peer address of TcpStream { addr: V4(172.26.0.3:45240), fd: 38 }: Os { code: 107, kind: NotConnected, message: "Transport endpoint is not connected" }
stacks-blockchain_1  | ERROR [1587722506.856] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:237] Failed to connect to peer 35.245.47.179:18444: connection to peer could not be (re-)established
stacks-blockchain_1  | ERROR [1587722523.407] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:237] Failed to connect to peer 35.245.47.179:18444: connection to peer could not be (re-)established
stacks-blockchain_1  | WARN [1587722523.407] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:248] Connection broken; retrying in 18.362285358864273 sec...
stacks-blockchain_1  | WARN [1587722542.830] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/burnchain.rs:799] Detected burnchain reorg at height 1252 (< 1267). Re-sync'ing...
stacks-blockchain_1  | ERROR [1587722542.831] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/burnchains/bitcoin_regtest_controller.rs:138] Unable to sync burnchain: Invalid argument (os error 22)
stacks-blockchain_1  | thread 'main' panicked at 'explicit panic', /usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/burnchains/bitcoin_regtest_controller.rs:139:17
```
**Environment (please complete the following information):**
* Latest core-node master (as of writing this issue) https://github.com/blockstack/stacks-blockchain/commit/6356b41e0e8e7115204cac43450ff56ea3ceec51
* One node running on MacOS, the other running on Ubuntu

**Comment by zone117x** [comment]
Looks like potential dupe of https://github.com/blockstack/stacks-blockchain/issues/1484

**Comment by zone117x** [comment]
Crash with debug and backtrace enabled:
```
stacks-blockchain_1  | 2020-04-24T11:39:24.252392041Z DEBUG [1587728364.252] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:514] Close HTTP connection on event 223
stacks-blockchain_1  | 2020-04-24T11:39:24.580755076Z DEBUG [1587728364.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:1635] local.80000000://V4(0.0.0.0:20444): network work state is NeighborWalk
stacks-blockchain_1  | 2020-04-24T11:39:24.580995399Z DEBUG [1587728364.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1584] local.80000000://V4(0.0.0.0:20444): walk state is HandshakeBegin
stacks-blockchain_1  | 2020-04-24T11:39:24.581214836Z DEBUG [1587728364.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:360] Walk clear state
stacks-blockchain_1  | 2020-04-24T11:39:24.581241670Z DEBUG [1587728364.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1662] local.80000000://V4(0.0.0.0:20444): walk has taken 118 steps (total of 546 walks)
stacks-blockchain_1  | 2020-04-24T11:39:24.581288000Z DEBUG [1587728364.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2077] >>>>>>>>>>>>>>>>>>>>>>> Begin Poll >>>>>>>>>>>>>>>>>>>>>>>>>>>>
stacks-blockchain_1  | 2020-04-24T11:39:24.581299772Z DEBUG [1587728364.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2079] <<<<<<<<<<<<<<<<<<<<<<<< End Poll <<<<<<<<<<<<<<<<<<<<<<<<<<<<
stacks-blockchain_1  | 2020-04-24T11:39:24.581462567Z DEBUG [1587728364.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:161] HTTP: Have 0 peers now (max 1000) inbound=true, including 0 from host of V4(192.168.64.4:60916)
stacks-blockchain_1  | 2020-04-24T11:39:24.581491443Z DEBUG [1587728364.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:192] Registered HTTP TcpStream { addr: V4(192.168.64.2:20443), peer: V4(192.168.64.4:60916), fd: 4 } as event 224 (outbound=None)
stacks-blockchain_1  | 2020-04-24T11:39:24.582715117Z DEBUG [1587728364.582] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:2838] No staging blocks
stacks-blockchain_1  | 2020-04-24T11:39:24.583557080Z DEBUG [1587728364.582] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:3034] No more staging blocks -- processed 0 in total
stacks-blockchain_1  | 2020-04-24T11:39:24.585907561Z DEBUG [1587728364.585] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/relay.rs:828] Remove all transactions beneath block height 237
stacks-blockchain_1  | 2020-04-24T11:39:24.913091540Z DEBUG [1587728364.912] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:1635] local.80000000://V4(0.0.0.0:20444): network work state is NeighborWalk
stacks-blockchain_1  | 2020-04-24T11:39:24.913146354Z DEBUG [1587728364.912] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1584] local.80000000://V4(0.0.0.0:20444): walk state is HandshakeBegin
stacks-blockchain_1  | 2020-04-24T11:39:24.913411391Z DEBUG [1587728364.913] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:360] Walk clear state
stacks-blockchain_1  | 2020-04-24T11:39:24.914072424Z DEBUG [1587728364.913] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1662] local.80000000://V4(0.0.0.0:20444): walk has taken 119 steps (total of 546 walks)
stacks-blockchain_1  | 2020-04-24T11:39:24.914085292Z DEBUG [1587728364.913] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2077] >>>>>>>>>>>>>>>>>>>>>>> Begin Poll >>>>>>>>>>>>>>>>>>>>>>>>>>>>
stacks-blockchain_1  | 2020-04-24T11:39:24.914094060Z DEBUG [1587728364.913] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2079] <<<<<<<<<<<<<<<<<<<<<<<< End Poll <<<<<<<<<<<<<<<<<<<<<<<<<<<<
stacks-blockchain_1  | 2020-04-24T11:39:24.914988369Z DEBUG [1587728364.914] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:2838] No staging blocks
stacks-blockchain_1  | 2020-04-24T11:39:24.915661551Z DEBUG [1587728364.915] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:3034] No more staging blocks -- processed 0 in total
stacks-blockchain_1  | 2020-04-24T11:39:24.917880953Z DEBUG [1587728364.917] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/relay.rs:828] Remove all transactions beneath block height 237
stacks-blockchain_1  | 2020-04-24T11:39:24.920897585Z DEBUG [1587728364.920] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:514] Close HTTP connection on event 224
stacks-blockchain_1  | 2020-04-24T11:39:25.250697973Z DEBUG [1587728365.250] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:1635] local.80000000://V4(0.0.0.0:20444): network work state is NeighborWalk
stacks-blockchain_1  | 2020-04-24T11:39:25.250741130Z DEBUG [1587728365.250] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1584] local.80000000://V4(0.0.0.0:20444): walk state is HandshakeBegin
stacks-blockchain_1  | 2020-04-24T11:39:25.251197798Z DEBUG [1587728365.251] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:360] Walk clear state
stacks-blockchain_1  | 2020-04-24T11:39:25.251222774Z DEBUG [1587728365.251] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1662] local.80000000://V4(0.0.0.0:20444): walk has taken 120 steps (total of 546 walks)
stacks-blockchain_1  | 2020-04-24T11:39:25.251427798Z DEBUG [1587728365.251] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2077] >>>>>>>>>>>>>>>>>>>>>>> Begin Poll >>>>>>>>>>>>>>>>>>>>>>>>>>>>
stacks-blockchain_1  | 2020-04-24T11:39:25.251757138Z DEBUG [1587728365.251] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2079] <<<<<<<<<<<<<<<<<<<<<<<< End Poll <<<<<<<<<<<<<<<<<<<<<<<<<<<<
stacks-blockchain_1  | 2020-04-24T11:39:25.251903098Z DEBUG [1587728365.251] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:161] HTTP: Have 0 peers now (max 1000) inbound=true, including 0 from host of V4(192.168.64.4:60918)
stacks-blockchain_1  | 2020-04-24T11:39:25.252069691Z DEBUG [1587728365.251] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:192] Registered HTTP TcpStream { addr: V4(192.168.64.2:20443), peer: V4(192.168.64.4:60918), fd: 4 } as event 225 (outbound=None)
stacks-blockchain_1  | 2020-04-24T11:39:25.252677914Z DEBUG [1587728365.252] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:2838] No staging blocks
stacks-blockchain_1  | 2020-04-24T11:39:25.253046958Z DEBUG [1587728365.252] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:3034] No more staging blocks -- processed 0 in total
stacks-blockchain_1  | 2020-04-24T11:39:25.254959117Z DEBUG [1587728365.254] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/relay.rs:828] Remove all transactions beneath block height 237
stacks-blockchain_1  | 2020-04-24T11:39:25.581129818Z DEBUG [1587728365.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:1635] local.80000000://V4(0.0.0.0:20444): network work state is NeighborWalk
stacks-blockchain_1  | 2020-04-24T11:39:25.581174127Z DEBUG [1587728365.580] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1584] local.80000000://V4(0.0.0.0:20444): walk state is HandshakeBegin
stacks-blockchain_1  | 2020-04-24T11:39:25.581545504Z DEBUG [1587728365.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:360] Walk clear state
stacks-blockchain_1  | 2020-04-24T11:39:25.581575825Z DEBUG [1587728365.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1662] local.80000000://V4(0.0.0.0:20444): walk has taken 121 steps (total of 546 walks)
stacks-blockchain_1  | 2020-04-24T11:39:25.581801007Z DEBUG [1587728365.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2077] >>>>>>>>>>>>>>>>>>>>>>> Begin Poll >>>>>>>>>>>>>>>>>>>>>>>>>>>>
stacks-blockchain_1  | 2020-04-24T11:39:25.581822906Z DEBUG [1587728365.581] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2079] <<<<<<<<<<<<<<<<<<<<<<<< End Poll <<<<<<<<<<<<<<<<<<<<<<<<<<<<
stacks-blockchain_1  | 2020-04-24T11:39:25.583043290Z DEBUG [1587728365.582] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:2838] No staging blocks
stacks-blockchain_1  | 2020-04-24T11:39:25.583677554Z DEBUG [1587728365.583] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:3034] No more staging blocks -- processed 0 in total
stacks-blockchain_1  | 2020-04-24T11:39:25.585589380Z DEBUG [1587728365.585] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/relay.rs:828] Remove all transactions beneath block height 237
stacks-blockchain_1  | 2020-04-24T11:39:25.587912409Z DEBUG [1587728365.587] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:514] Close HTTP connection on event 225
stacks-blockchain_1  | 2020-04-24T11:39:25.916025587Z DEBUG [1587728365.915] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:1635] local.80000000://V4(0.0.0.0:20444): network work state is NeighborWalk
stacks-blockchain_1  | 2020-04-24T11:39:25.916065886Z DEBUG [1587728365.915] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1584] local.80000000://V4(0.0.0.0:20444): walk state is HandshakeBegin
stacks-blockchain_1  | 2020-04-24T11:39:25.917053738Z DEBUG [1587728365.916] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:360] Walk clear state
stacks-blockchain_1  | 2020-04-24T11:39:25.917093628Z DEBUG [1587728365.916] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1662] local.80000000://V4(0.0.0.0:20444): walk has taken 122 steps (total of 546 walks)
stacks-blockchain_1  | 2020-04-24T11:39:25.917100820Z DEBUG [1587728365.916] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2077] >>>>>>>>>>>>>>>>>>>>>>> Begin Poll >>>>>>>>>>>>>>>>>>>>>>>>>>>>
stacks-blockchain_1  | 2020-04-24T11:39:25.917107233Z DEBUG [1587728365.916] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2079] <<<<<<<<<<<<<<<<<<<<<<<< End Poll <<<<<<<<<<<<<<<<<<<<<<<<<<<<
stacks-blockchain_1  | 2020-04-24T11:39:25.917112818Z DEBUG [1587728365.916] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:161] HTTP: Have 0 peers now (max 1000) inbound=true, including 0 from host of V4(192.168.64.4:60920)
stacks-blockchain_1  | 2020-04-24T11:39:25.917117233Z DEBUG [1587728365.916] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:192] Registered HTTP TcpStream { addr: V4(192.168.64.2:20443), peer: V4(192.168.64.4:60920), fd: 4 } as event 226 (outbound=None)
stacks-blockchain_1  | 2020-04-24T11:39:25.917832250Z DEBUG [1587728365.917] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:2838] No staging blocks
stacks-blockchain_1  | 2020-04-24T11:39:25.918495903Z DEBUG [1587728365.917] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:3034] No more staging blocks -- processed 0 in total
stacks-blockchain_1  | 2020-04-24T11:39:25.921319850Z DEBUG [1587728365.920] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/relay.rs:828] Remove all transactions beneath block height 237
stacks-blockchain_1  | 2020-04-24T11:39:26.107687942Z DEBUG [1587728366.107] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:193] Begin peer handshake to 35.245.47.179:18444
stacks-blockchain_1  | 2020-04-24T11:39:26.107732036Z DEBUG [1587728366.107] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:283] Send version (nonce=16575575296942416477) to 35.245.47.179:18444
stacks-blockchain_1  | 2020-04-24T11:39:26.144910049Z DEBUG [1587728366.144] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:291] Handle version -- remote peer blockchain height is 0
stacks-blockchain_1  | 2020-04-24T11:39:26.144945526Z DEBUG [1587728366.144] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:306] Send verack
stacks-blockchain_1  | 2020-04-24T11:39:26.145248404Z DEBUG [1587728366.145] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:315] Handle verack
stacks-blockchain_1  | 2020-04-24T11:39:26.145994051Z DEBUG [1587728366.145] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/network.rs:201] Established connection to 35.245.47.179:18444, who has 0 blocks
stacks-blockchain_1  | 2020-04-24T11:39:26.146038657Z DEBUG [1587728366.145] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/spv.rs:413] Have all headers up to 1010
stacks-blockchain_1  | 2020-04-24T11:39:26.146045361Z DEBUG [1587728366.145] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/indexer.rs:371] begin_session() terminates conversation
stacks-blockchain_1  | 2020-04-24T11:39:26.148030718Z DEBUG [1587728366.147] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/bitcoin/indexer.rs:543] Chain history is consistent up to 1011
stacks-blockchain_1  | 2020-04-24T11:39:26.148051636Z WARN [1587728366.147] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/burnchains/burnchain.rs:799] Detected burnchain reorg at height 1011 (< 1026). Re-sync'ing...
stacks-blockchain_1  | 2020-04-24T11:39:26.149008119Z ERROR [1587728366.148] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/burnchains/bitcoin_regtest_controller.rs:138] Unable to sync burnchain: Invalid argument (os error 22)
stacks-blockchain_1  | 2020-04-24T11:39:26.149029557Z thread 'main' panicked at 'explicit panic', /usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/burnchains/bitcoin_regtest_controller.rs:139:17
stacks-blockchain_1  | 2020-04-24T11:39:26.149044381Z stack backtrace:
stacks-blockchain_1  | 2020-04-24T11:39:26.254180471Z DEBUG [1587728366.253] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:1635] local.80000000://V4(0.0.0.0:20444): network work state is NeighborWalk
stacks-blockchain_1  | 2020-04-24T11:39:26.254220021Z DEBUG [1587728366.253] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1584] local.80000000://V4(0.0.0.0:20444): walk state is HandshakeBegin
stacks-blockchain_1  | 2020-04-24T11:39:26.254482232Z DEBUG [1587728366.254] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:360] Walk clear state
stacks-blockchain_1  | 2020-04-24T11:39:26.254505951Z DEBUG [1587728366.254] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/neighbors.rs:1662] local.80000000://V4(0.0.0.0:20444): walk has taken 123 steps (total of 546 walks)
stacks-blockchain_1  | 2020-04-24T11:39:26.254754017Z DEBUG [1587728366.254] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2077] >>>>>>>>>>>>>>>>>>>>>>> Begin Poll >>>>>>>>>>>>>>>>>>>>>>>>>>>>
stacks-blockchain_1  | 2020-04-24T11:39:26.254984489Z DEBUG [1587728366.254] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/p2p.rs:2079] <<<<<<<<<<<<<<<<<<<<<<<< End Poll <<<<<<<<<<<<<<<<<<<<<<<<<<<<
stacks-blockchain_1  | 2020-04-24T11:39:26.255965091Z DEBUG [1587728366.255] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:2838] No staging blocks
stacks-blockchain_1  | 2020-04-24T11:39:26.256710031Z DEBUG [1587728366.256] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/chainstate/stacks/db/blocks.rs:3034] No more staging blocks -- processed 0 in total
stacks-blockchain_1  | 2020-04-24T11:39:26.258878807Z DEBUG [1587728366.258] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/relay.rs:828] Remove all transactions beneath block height 237
stacks-blockchain_1  | 2020-04-24T11:39:26.262489398Z DEBUG [1587728366.262] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/src/net/server.rs:514] Close HTTP connection on event 226
stacks-blockchain_1  | 2020-04-24T11:39:26.328366840Z    0: backtrace::backtrace::libunwind::trace
stacks-blockchain_1  | 2020-04-24T11:39:26.328408273Z              at ./cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/libunwind.rs:88
stacks-blockchain_1  | 2020-04-24T11:39:26.328414825Z    1: backtrace::backtrace::trace_unsynchronized
stacks-blockchain_1  | 2020-04-24T11:39:26.328419110Z              at ./cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/mod.rs:66
stacks-blockchain_1  | 2020-04-24T11:39:26.328424552Z    2: std::sys_common::backtrace::_print_fmt
stacks-blockchain_1  | 2020-04-24T11:39:26.328429704Z              at src/libstd/sys_common/backtrace.rs:77
stacks-blockchain_1  | 2020-04-24T11:39:26.328433876Z    3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
stacks-blockchain_1  | 2020-04-24T11:39:26.328452043Z              at src/libstd/sys_common/backtrace.rs:59
stacks-blockchain_1  | 2020-04-24T11:39:26.338712018Z    4: core::fmt::write
stacks-blockchain_1  | 2020-04-24T11:39:26.338950996Z              at src/libcore/fmt/mod.rs:1052
stacks-blockchain_1  | 2020-04-24T11:39:26.339340329Z    5: std::io::Write::write_fmt
stacks-blockchain_1  | 2020-04-24T11:39:26.339543267Z              at src/libstd/io/mod.rs:1426
stacks-blockchain_1  | 2020-04-24T11:39:26.340211340Z    6: std::sys_common::backtrace::_print
stacks-blockchain_1  | 2020-04-24T11:39:26.340519704Z              at src/libstd/sys_common/backtrace.rs:62
stacks-blockchain_1  | 2020-04-24T11:39:26.340973116Z    7: std::sys_common::backtrace::print
stacks-blockchain_1  | 2020-04-24T11:39:26.341171192Z              at src/libstd/sys_common/backtrace.rs:49
stacks-blockchain_1  | 2020-04-24T11:39:26.341887427Z    8: std::panicking::default_hook::{{closure}}
stacks-blockchain_1  | 2020-04-24T11:39:26.342193568Z              at src/libstd/panicking.rs:204
stacks-blockchain_1  | 2020-04-24T11:39:26.342688931Z    9: std::panicking::default_hook
stacks-blockchain_1  | 2020-04-24T11:39:26.342989900Z              at src/libstd/panicking.rs:224
stacks-blockchain_1  | 2020-04-24T11:39:26.343385152Z   10: std::panicking::rust_panic_with_hook
stacks-blockchain_1  | 2020-04-24T11:39:26.343667255Z              at src/libstd/panicking.rs:472
stacks-blockchain_1  | 2020-04-24T11:39:26.344282621Z   11: std::panicking::begin_panic
stacks-blockchain_1  | 2020-04-24T11:39:26.344558574Z              at ./rustc/b8cedc00407a4c56a3bda1ed605c6fc166655447/src/libstd/panicking.rs:399
stacks-blockchain_1  | 2020-04-24T11:39:26.345782172Z   12: stacks_node::burnchains::bitcoin_regtest_controller::BitcoinRegtestController::receive_blocks
stacks-blockchain_1  | 2020-04-24T11:39:26.346015327Z              at ./usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/burnchains/bitcoin_regtest_controller.rs:139
stacks-blockchain_1  | 2020-04-24T11:39:26.346719236Z   13: <stacks_node::burnchains::bitcoin_regtest_controller::BitcoinRegtestController as stacks_node::burnchains::BurnchainController>::sync
stacks-blockchain_1  | 2020-04-24T11:39:26.346944940Z              at ./usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/burnchains/bitcoin_regtest_controller.rs:516
stacks-blockchain_1  | 2020-04-24T11:39:26.348949949Z   14: stacks_node::run_loop::neon::RunLoop::start
stacks-blockchain_1  | 2020-04-24T11:39:26.349137229Z              at ./usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/run_loop/neon.rs:70
stacks-blockchain_1  | 2020-04-24T11:39:26.350814506Z   15: stacks_node::main
stacks-blockchain_1  | 2020-04-24T11:39:26.351084255Z              at ./usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/6356b41/testnet/src/main.rs:77
stacks-blockchain_1  | 2020-04-24T11:39:26.351947790Z   16: std::rt::lang_start::{{closure}}
stacks-blockchain_1  | 2020-04-24T11:39:26.352252128Z              at ./rustc/b8cedc00407a4c56a3bda1ed605c6fc166655447/src/libstd/rt.rs:67
stacks-blockchain_1  | 2020-04-24T11:39:26.353005952Z   17: std::rt::lang_start_internal::{{closure}}
stacks-blockchain_1  | 2020-04-24T11:39:26.353319540Z              at src/libstd/rt.rs:52
stacks-blockchain_1  | 2020-04-24T11:39:26.353843159Z   18: std::panicking::try::do_call
stacks-blockchain_1  | 2020-04-24T11:39:26.354062011Z              at src/libstd/panicking.rs:305
stacks-blockchain_1  | 2020-04-24T11:39:26.354377380Z   19: __rust_maybe_catch_panic
stacks-blockchain_1  | 2020-04-24T11:39:26.354488156Z              at src/libpanic_unwind/lib.rs:86
stacks-blockchain_1  | 2020-04-24T11:39:26.354802895Z   20: std::panicking::try
stacks-blockchain_1  | 2020-04-24T11:39:26.354961255Z              at src/libstd/panicking.rs:281
stacks-blockchain_1  | 2020-04-24T11:39:26.355198205Z   21: std::panic::catch_unwind
stacks-blockchain_1  | 2020-04-24T11:39:26.355330760Z              at src/libstd/panic.rs:394
stacks-blockchain_1  | 2020-04-24T11:39:26.355572488Z   22: std::rt::lang_start_internal
stacks-blockchain_1  | 2020-04-24T11:39:26.355752815Z              at src/libstd/rt.rs:51
stacks-blockchain_1  | 2020-04-24T11:39:26.356094409Z   23: std::rt::lang_start
stacks-blockchain_1  | 2020-04-24T11:39:26.356261075Z              at ./rustc/b8cedc00407a4c56a3bda1ed605c6fc166655447/src/libstd/rt.rs:67
stacks-blockchain_1  | 2020-04-24T11:39:26.356541847Z   24: main
stacks-blockchain_1  | 2020-04-24T11:39:26.356770599Z   25: __libc_start_main
stacks-blockchain_1  | 2020-04-24T11:39:26.357087221Z   26: _start
```

**Comment by Trudko** [comment]
@zone117x  have a same issue, I don't think it's related to #1484. AFAIK that should have been already fixed on master branch.

**Comment by zone117x** [comment]
This bug is public enemy num. 1 right now for sidecar and faucet

**Comment by jcnelson** [comment]
Did the Neon master node reboot while you were running this?  That could be a cause of the perceived bitcoin reorg.

**Comment by jcnelson** [comment]
I think #1472 might be part of the fix here.  Stacks 2.0 uses the same strategy for accumulating SPV headers as Stacks 1.0, and it's caused problems on Mac OS before.

**Comment by jcnelson** [comment]
Should be fixed in #1522.  Going to go ahead and close this issue; please re-open if problem persists.  Thanks for the bug report!
